src/test/java/org/cactoos/iterator/NoNullsTest.java

```diff
@@ -75,20 +67,20 @@ public String toString() {
         ).affirm();
     }
 
-    /*
-    * @todo #1039:15min Currently it's impossible to match error messages
-    *  by a pattern or partially. Replace {@link Rule} with {@link Throws} after
-    *  <a href="https://github.com/llorllale/cactoos-matchers/issues/108">llorllale/cactoos-matchers#108</a>
-    *  is fixed
-    */
     @Test
     public void nthThrowsErrorIfNull() {
-        this.exception.expect(IllegalStateException.class);
-        final Iterator<String> itr = new NoNulls<>(
-            new IteratorOf<>("a", "b", null, "c")
-        );
-        while (itr.hasNext()) {
-            itr.next();
-        }
+        new Assertion<>(
+            "nth throws error if null",
+            () -> {
+                final Iterator<String> itr = new NoNulls<>(
+                    new IteratorOf<>("a", "b", null, "c")
+                );
+                while (itr.hasNext()) {
+                    itr.next();
+                }
+                return null;
```

Comment:
I'm not sure about this `return null;`, but I have no better solution

Reply:
@iakunin you can't use anything else? Like an `Object`? If I understood correctly, this `return null` is never reached. Am I right?

Reply:
@fabriciofx of course I can use an `Object`. Yes, `return null` is never reached.

Reply:
@fabriciofx of course I can use an Object. Yes, return null is never reached.